app/services/ping.rb

```diff
@@ -36,11 +36,11 @@ def statuses
     private
 
     def duration_ms(start)
-      ((Time.new - start) * 1000).round.to_s
+      ((Process.clock_gettime(Process::CLOCK_MONOTONIC) - start) * 1000).round.to_s
```

Comment:
This made me look up what it returns. Turns out there is another optional argument for the unit. So this should work:
```suggestion
      (Process.clock_gettime(Process::CLOCK_MONOTONIC, :millisecond) - start).to_s
```

Now I'm wondering if it makes sense to introduce a helper for this.

Reply:
I debated the same, since you've mentioned it I will add one.